bpaotu/bpaotu/frontend/src/components/samples_map.tsx

```diff
@@ -32,6 +32,34 @@ const ArcGIS = {
     '&amp;copy; i-cubed, USDA, USGS, AEX, GeoEye, Getmapping, Aerogrid, IGN, IGP, UPR-EGP, and the GIS User Community'
 }
 
+// tslint:disable-next-line:max-classes-per-file 
+class BPAImages extends React.Component<any, any> {
+    constructor(props: any) {
+        super(props)
+
+        const imgs = []
+        if (typeof this.props.imgUrls !== 'undefined') {
+            for (let i=0; i<Object.keys(this.props.imgUrls).length; i++) {
+                imgs.push(<img src={`/process_img/${this.props.lat}/${this.props.lng}/${i}`} />)
+            }
+        }
+
+        this.state = {
+            img: imgs
+        }
+    }
+
+    public render() {
+        return (
+            <div>
+                {this.state.img}
+            </div>
+        )
+    }
+}
+
+
+// tslint:disable-next-line:max-classes-per-file
 class BPASamples extends React.Component<any, any> {
```

Comment:
again, don't disable the linter, make a new file

Reply:
Following @sztamas suggestion.